### ISSUE TITLE
Nether Block Lag on Single Player World

### ISSUE THREAD

**Issue by UnlimitedVoice** [Sunekaer/stoneBlock#468]
## Bug Report
Whenever I go into the Nether, and only in the Nether, I experience an INSANE amount of block lag. To the point of it being unplayable. Is there any way to fix this? In the video provided when I go into the nether, I open the F3 menu so you can see everything.

## Steps to Reproduce (for bugs)
1. This is a Youtube video I made to show what exactly I am experiencing. I apologize for the grainy quality.
2. https://youtu.be/_NF7CrblFLI

## Logs
* Client/Server Log: https://pastebin.com/raw/9XmP0KKB
* Crash Log: No Crashes

## Client Information
* Modpack Version: 1.0.25
* Server/LAN/Single Player: Single Player
* Optifine Installed: No
* Shaders Enabled: No

## System Specifications
* AMD Ryzen 7 1800X 8-Core Processor (3.6GHz)
* TridentZ 32GB RGB RAM (12GB Allocated to Minecraft
* NVIDIA GeForce GTX 1060 6GB with Drivers up-to-date
* Gigabyte AX370-Gaming Motherboard

**Comment by Sunekaer** [comment]
Have you changed any config?
i cant recreate this problem, try deleting the nether from the world folder and see if that helps

**Comment by UnlimitedVoice** [comment]
So... I went ahead and started a new world and played up until I got to the Nether. I just beligned to get to the Nether. When I went into the Nether, everything was fine. It all loaded correctly and there was no lag. So all I can think of is that the particular world that was giving me trouble was somehow corrupted or something to that effect. I never changed any config by the way, it was my second world loaded, after having my first work be a test to see what the pack was about. And I didn’t delete the Nether folder. Again - I think it had something to do with corruption.

Thanks for the advice though, I appreciate it!
